server/src/main/proto/spine/server/server_environment.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package spine.server;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.server";
+option java_outer_classname = "ServerEnvironmentProto";
+option java_multiple_files = true;
+
+// A type of the environment system is deployed to.
+//
+enum DeploymentType {
+
+    // Running on a unrecognized environment.
```

Comment:
We run in a standalone mode. Meaning that no specific runtime deployment is recognized.